package-lock.json

```diff
@@ -3833,8 +3822,7 @@
     "util-deprecate": {
       "version": "1.0.2",
       "resolved": "https://registry.npmjs.org/util-deprecate/-/util-deprecate-1.0.2.tgz",
-      "integrity": "sha1-RQ1Nyfpw3nMnYvvS1KKJgUGaDM8=",
-      "dev": true
+      "integrity": "sha1-RQ1Nyfpw3nMnYvvS1KKJgUGaDM8="
```

Comment:
missing optional or removed?

Reply:
The change for these is just that the `"dev": true` line was dropped. This was a result of dropping `keytar` as a `devDependency`. So the ones that changed are no longer part of a dev dependency tree.